content/pandas/concepts/groupby/terms/sum/sum.md

```diff
@@ -57,3 +57,20 @@ This example produces the following output:
 1   B   13.0
 2   C   12.0
 ```
+
+### Try It Yourself
```

Comment:
```suggestion
## Codebyte Example
```

Reply:
I believe "try it yourself" is the tag line used on another site. We generally stick to "Codebyte Example"